### DISCUSSION TITLE
Is it possible to change a project s [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2874] [Discord Threads]
> 🙋‍♂️ **brunoqc**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Acedia**  
#### Unfortunately we only allow editing of project title for now, you have to recreate to edit connection details  
#  
> 🙋‍♂️ **brunoqc**  
#### okay, It's not a big deal. It's only 1 table. Thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999399956240674909)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
